app/controllers/posts_controller.rb

```diff
@@ -5,9 +5,9 @@ def index
   end
 
   def show
-    @post = Post.includes(:author, :comments, :likes).find(params[:id])
+    @post = Post.find(params[:id])
     @user = User.find(params[:user_id])
-    @comments = @post.comments
+    @comments = @post.comments.includes(:author)
   end
```

Comment:
- [ ] It is a requirement that you use `CanCanCan` for Post authorization in `Post Controller`, kindly do so
![image](https://user-images.githubusercontent.com/81932478/207952449-129c09aa-cbd2-49d6-8460-35eff577b3b6.png)

Reply:
This change has been done well